### ISSUE TITLE
Links missing a text alternate

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#628]
Best practice - A

Please remove empty anchors from links, 132 occurrences, details here - https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?pageId=38246421766&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r11/failed/KzYxY3pBOFc=;NTUzYWQ2ZTc=
